offchain/create_invoice.js

```diff
@@ -12,6 +12,7 @@ const {parseAmount} = require('ln-accounting');
 const {parsePaymentRequest} = require('ln-service');
 const {returnResult} = require('asyncjs-util');
 const {subscribeToForwardRequests} = require('ln-service');
+const moment = require('moment');
```

Comment:
i think this isn't needed if native Date is used

Reply:
Removed